src/components/UserCard/UserCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styles from './UserCard.module.css';
+
+interface Details {
+  key: any;
+  firstName: any;
+  lastName: any;
+  image: any;
+}
```

Comment:
Please use a descriptive name for your interfaces. `Details` does not indicate the purpose of this interface.

See the suggestion below
```suggestion
interface UserCardProps {
  key: any;
  firstName: any;
  lastName: any;
  image: any;
}
```